work11/index.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+const div = document.querySelector(".wrap")
+const ul = document.getElementById("js-list");
+const url = "https://jsondata.okiba.me/v1/json/d4ex5210822124352"
+
+function getData() {
+  const result = new Promise((resolve, reject) => {
+    setTimeout(() => {
+      resolve(
+        async function a() {
```

Comment:
Do you have to give it a name to this function? I didn't see anywhere you're calling the function called 'a'.

Reply:
Changed function name a to anonymous function.

Please check


commit number
1dc5e4a